content/en/users/tutorials/data-transfer-object-storage/_index.md

```diff
@@ -1,11 +1,168 @@
+necessary. To install the latest version please add the
+[FTS3 Production](https://fts-repo.web.cern.ch/fts-repo/fts3-prod-el7.repo)
+repository to your configuration and update the client{{% /alert %}}
+
+## Prerequisites
+
+As first step please make sure that you have installed the FTS client as
+described in [Data Transfer](../../data-transfer/), and in particular
+[Clients](../../data-transfer/clients/) for the command-line FTS and to have
+your certificate installed in your browser to use
+[WebFTS](../../data-transfer/webfts/) browser based client.
+
+To access services and resources in the
+[EGI Federated Cloud](../../getting-started), you will need:
+
+- An [EGI Check-in](../../check-in) account, you can
+  [sign up here](../../check-in/signup)
+- Enrollment into a [Virtual Organisation](../../check-in/vos) (VO) that has
+  access to the services and resources you need
+- An [Object Storage](../../online-storage/object-storage/) for which you need
+  to have all the credentials available (any S3 compatible storage should work)
+- Permission to add the Object Storage credential to the FTS server or
+  alternatively for this operation you may contact support support **at**
+  egi.eu.
+
+## FTS client usage
+
+### Step 1 Configuration check and Proxy creation
+
+For this two steps please refer to the
+["Data transfer with grid storage"](../data-transfer-grid-storage/) tutorial.
+
+### Step 2 Find the storage
+
+As for the "Data transfer with grid storage" tutorial you can look for the
+available storage on
+[VAPOR service](https://operations-portal.egi.eu/vapor/resources/GL2ResVO) while
+the Object Store can be one created as described in the
+[Object Storage](../../online-storage/object-storage/) section or trough a
+provider such as Amazon, Azure, etc
+
+### Step 3 Add the Object Storage credential to the FTS server
+
+Following is an example of the command that can be used to add the Object Store
+credential to the FTS server. The fist step is to register the Object Storage.
+The name of the storage, is S3: + the domain part of the URL (for example
+`https://s3.cl2.du.cesnet.cz` -> `S3:s3.cl2.du.cesnet.cz`)
+
+```shell
+$ curl -E "${X509_USER_PROXY}" \
+  --cacert "${X509_USER_PROXY}" \
+  --capath "/etc/grid-security/certificates" \
+  https://fts3devel01.cern.ch:8446/config/cloud_storage \
+  -H "Content-Type: application/json" \
+  -X POST \
+  -d '{"storage_name":"S3:s3.cl2.du.cesnet.cz"}'
+```
+
+And then, add the keys, so the requests can be signed.
+
+```shell
+$ curl -E "${X509_USER_PROXY}" \
+  --cacert "${X509_USER_PROXY}" \
+  --capath "/etc/grid-security/certificates" \
+  "https://fts3devel01.cern.ch:8446/config/cloud_storage/S3:s3.cl2.du.cesnet.cz" \
+  -H "Content-Type: application/json" \
+  -X POST \
+  -d "${config}"
+```
+
+Where `config` is a JSON message with the following content:
+
+```json
+{
+  "vo_name": "/dteam/Role=NULL/Capability=NULL",
+  "access_key": "ACCESS_KEY",
+  "secret_key": "SECRET_KEY"
+}
+```
```

Comment:
The usage of the `config` stuff is a bit ambiguous and may confuse the users. It may be easier to either include the proper message in the command or to make use of the `curl` feature allowing to read the JSON message from a file.
Like using `-d @config.json` or `--data-binary @config.json` depending on what is needed/the best/working. See man curl(1).

Reply:
updated with --data  in curl and name `config.json` for the JSON configuration